chroma-dependencies/Makefile

```diff
@@ -21,7 +21,7 @@ cleanrepo: TARGET=cleanrepo
 requirements::
 	# these are the ones provided by EL and EPEL and our own repo
 	set -e;                                                   \
-	{ echo "../chroma-externals/netaddr-0.7.5.zip";           \
+	{ echo "https://github.com/drkjam/netaddr/archive/rel-0.7.5.tar.gz"; \
```

Comment:
Because I typically use an 80 character terminal and the couple of exceptions are still obvious but lots of whitespace with line continuators wrapped is not at all obvious.